content/300-guides/200-deployment/201-serverless/500-deploy-to-netlify.mdx

```diff
@@ -128,14 +77,14 @@ netlify deploy
 
 You can now test the deployed application.
 
-## Summary
+## Connection pooling
 
-You have successfully configured continuous deployments and integration with Netlify for an application that uses Prisma.
+When you use a Function-as-a-Service provider, like Netlify, it is beneficial to pool database connections for performance reasons. This is because every function invocation may result in a new connection to your database which can quickly run out of open connections.
 
-Every time you push changes to the `main` branch of the repository, Netlify starts a new production build and deployment. If you push changes to another branch, Netlify starts preview builds and deployments.
+You can use [Accelerate](/data-platform/accelerate) for connection pooling, to reduce your Prisma Client bundle size, and to avoid cold starts.
 
-When you use a FaaS (function as a service) environment to interact with a database, it is beneficial to pool database connections for performance reasons. This is because every function invocation may result in a new connection to the database (this is not a problem with a constantly running node.js server).
+For more information on connection management for serverless environments, refer to our [connection management guide](/guides/performance-and-optimization/connection-management#serverless-environments-faas).
 
-You can use the [Data Proxy](/data-platform/classic-projects/data-proxy) for connection pooling, to reduce your Prisma Client bundle size, and to avoid cold starts.
+## Summary
 
-For more information on the rest of the existing solutions, refer to the [connection management guide for serverless environments](/guides/performance-and-optimization/connection-management#serverless-environments-faas).
+You now have a successfully configured and deployed application using Prisma. Congratulations!
```

Comment:
I think we can do without this section. We could repurpose this section either in this PR or a later PR (preferably) to provide additional information that may be relevant.